tests/ui/perfherder/compare_table_test.jsx

```diff
@@ -132,6 +133,15 @@ const compareTable = (
     />,
   );
 
+const comparePageLabel = () =>
+  render(
+    <ComparePageLabel
```

Comment:
Nit: This is technically changing the page title. Labels usually used with inputs, so I think title would be clearer.

Reply:
Changed. Will be included in a new commit, along with the changes requested by this code review.